src/openms/include/OpenMS/CHEMISTRY/ModificationDefinition.h

```diff
@@ -63,8 +63,11 @@ namespace OpenMS
     /// copy constructor
     ModificationDefinition(const ModificationDefinition& rhs);
 
-    /// detailed constructor specifying the modifications name
-    explicit ModificationDefinition(const String& mod);
+    /// detailed constructor specifying the modification by name
+    explicit ModificationDefinition(const String& mod, bool fixed = true, UInt max_occur = 0);
+
+    /// detailed constructor specifying the modification by name
+    ModificationDefinition(const ResidueModification& mod, bool fixed = true, UInt max_occur = 0);
```

Comment:
Implicit conversion is considered one of the worst sides of c++ as it produces very hard to debug errors. Actually I think cpplint will also put out a warning about it.